pkg/types/validation/installconfig.go

```diff
@@ -369,8 +361,8 @@ func validateNetworking(n *types.Networking, singleNodeOpenShift bool, fldPath *
 		allErrs = append(allErrs, field.Invalid(fldPath.Child("networkType"), n.NetworkType, "networkType Kuryr is not supported on OpenShift later than 4.14"))
 	}
 
-	if singleNodeOpenShift && n.NetworkType == string(operv1.NetworkTypeOpenShiftSDN) {
-		allErrs = append(allErrs, field.Invalid(fldPath.Child("networkType"), n.NetworkType, "networkType OpenShiftSDN is currently not supported on Single Node OpenShift"))
+	if n.NetworkType == string(operv1.NetworkTypeOpenShiftSDN) {
+		allErrs = append(allErrs, field.Invalid(fldPath.Child("networkType"), n.NetworkType, "networkType OpenShiftSDN is deprecated, please use OVNKubernetes"))
 	}
```

Comment:
Isn't `deprecated` by definition means - still supported but support will be removed in future versions. This seems like wrong wording. It should say `Support for OpenshiftSDN is removed`.

Reply:
I think you are correct and it's a good point! I think to be on the safe side we should say something along the lines that is not supported at install time.